src/main/java/org/gridsuite/modification/server/entities/ModificationGroupEntity.java

```diff
@@ -28,6 +36,12 @@
     @Column(name = "id")
     private UUID id;
 
+    @OneToMany(fetch = FetchType.LAZY)
+    @JoinColumn(name = "groupId", foreignKey = @ForeignKey(name = "group_id_fk_constraint"), nullable = false)
+    @Setter
+    @OrderColumn
```

Comment:
With this approach we have as many update as elements in the list when we create a list of modifications and when we insert a list of modifications
Is this really what we want ?

Reply:
yes the other option is to use native sql, and only work for inserting